src/view/frontend/layout/hyvacheckout_checkout_index.xml

```diff
@@ -6,15 +6,21 @@
     </head>
     <body>
         <referenceContainer name="content">
+            <block class="Hyva\Checkout\Block\CheckoutTranslator" name="checkout.translations" template="Hyva_Checkout::translation.phtml">
+                <arguments>
+                    <argument name="checkout_translations" xsi:type="array">
+                        <item name="hyva_checkout_basic" xsi:type="string">
```

Comment:
why is this a comma separated string and not an array?

Reply:
it is a cumbersome process to add an `<item />` entry for each translatable string. If we think about the advantages, we can replace an individual string with an another value. But, the possibility of such usage is extremely rare. 

Passing a string is less code and we are deliberately passing them inside a group which allow us to provide additional translation string from an another module.

Reply:
We'll face the same issue here as Fabian mentioned, translations with a `,` will break. Maybe wrap them with quotes, like native Magento

Reply:
@schmengler @wigman we are now using `str_getcsv` function instead of `explode`. This will allow us to escape comma in the layout xml string literals. I have added a comment also in the layout XML file explaining how to do it.